### ISSUE TITLE
How to define schema for 204 No Content response?

### ISSUE THREAD

**Issue by playpauseandstop** [vitalik/django-ninja#91]
`django-ninja` allows to define multiple schemas for different response status codes via `response` dict such as,

```py
router = Router()

@router.post("/articles", response={201: schemas.Article})
def create_article(request: HttpRequest, payload: schemas.CreateArticle):
    ...
```

However I'm not sure, how to document `204 No Content` response with `django-ninja`.

I tried to pass `204: None` to `response` definition dict but it results in,

```
  File "./.venv/lib/python3.9/site-packages/ninja/openapi/schema.py", line 66, in operation_details
    "responses": self.responses(operation),
  File "./.venv/lib/python3.9/site-packages/ninja/openapi/schema.py", line 171, in responses
    schema, _ = self._create_schema_from_model(model, by_alias=False)
  File "./.venv/lib/python3.9/site-packages/ninja/openapi/schema.py", line 117, in _create_schema_from_model
    schema = model_schema(model, ref_prefix=REF_PREFIX, by_alias=by_alias)
  File "pydantic/schema.py", line 149, in pydantic.schema.model_schema
  File "pydantic/utils.py", line 262, in pydantic.utils.get_model
  File "~/.pyenv/versions/3.9.2/lib/python3.9/abc.py", line 102, in __subclasscheck__
    return _abc_subclasscheck(cls, subclass)
TypeError: issubclass() arg 1 must be a class
```

With that in mind, if there is a way to document 204 No Content response with `django-ninja`?

---

In `openapi.yaml` I manage to define 204 No Content responses as just,

```yaml
      responses:
        "204":
          description: "New email is same as current email"
```

**Comment by vitalik** [comment]
not sure I understood, maybe like this: 

```Python

class NoContent(Schema):
     description: str

@router.post("/articles", response={201: schemas.Article, 204: NoContent})
def create_article(request: HttpRequest, payload: schemas.CreateArticle):
   if True:
          return 204, {'description': 'Bla bla bla'}
   else:
          return 201, artibcle_object
    ...
```

**Comment by playpauseandstop** [comment]
[204 No Content](https://httpstatuses.com/204) is a special use case, when server do not respond with any content in body.

From [OpenAPI Documentation](https://swagger.io/docs/specification/describing-responses/),

> **Empty Response Body**
>
> Some responses, such as 204 No Content, have no body. To indicate the response body is empty, do not specify a content for the response:
>
>      responses:
>        '204':
>          description: The resource was deleted successfully.

So adding `NoContent` schema do not fix the issue.

**Comment by vitalik** [comment]
@playpauseandstop 
yeah, `response={204: None}` would make most sense here...


maybe for now this will work for you case

```Python
class Empty(Schema):
    pass


@api.post("/articles", response={201: xxx, 204: Empty})
def create_article(request....):
    return 204, ""
```

**Comment by vitalik** [comment]
implemented [in 0.12](https://django-ninja.rest-framework.com/tutorial/response-schema/#empty-responses)

**Comment by playpauseandstop** [comment]
Checked with `django-ninja==0.12.1`. All works as expected!

Thanks for quick fix!
